src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,17 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  if (typeof text === "string") {
```

Comment:
Wrapping your whole program in an `if else` would be problematic if the code were longer. Learn about `Early Return`

Reply:
Fixed. Thanks for pointing that out.